jobs/atc/spec

```diff
@@ -507,3 +507,38 @@ properties:
     description: |
       Client secret for CredHub authorization.
     default: ""
+
+  aws-ssm.aws_region:
```

Comment:
I think this should be `aws_ssm.region` (and the following properties should be changed similarly). Underscores are more conventional in BOSH and we removed the duplicate prefix in the CLI.